gcloud/storage/blob.py

```diff
@@ -109,6 +111,23 @@ def path_helper(bucket_path, blob_name):
         """
         return bucket_path + '/o/' + quote(blob_name, safe='')
 
+    @staticmethod
+    def _get_customer_encryption_headers(key):
+        """Builds customer encyrption key headers
+
+        :type key: str
+        :param key: 32 byte key to build request key and hash.
```

Comment:
Those both would be `None` after implementing https://github.com/GoogleCloudPlatform/gcloud-python/pull/1844#discussion_r66176613 ? Should I just leave them out in that case?